lib/standards/html-elms.js

```diff
@@ -350,14 +350,17 @@ const htmlElms = {
       },
       usemap: {
         matches: '[usemap]',
-        contentTypes: ['interactive', 'embedded', 'phrasing', 'flow']
+        contentTypes: ['interactive', 'embedded', 'flow']
```

Comment:
I wasn't sure if the content type needed to be removed from both the `default` and `usemap` objects - I'm not sure how usemap is used.

Reply:
Yep, removed from both should be fine.